core/app/[locale]/layout.tsx

```diff
@@ -76,7 +78,9 @@ export default function RootLayout({ children, params: { locale } }: Props) {
       <body className="flex h-screen min-w-[375px] flex-col">
         <Notifications />
         <NextIntlClientProvider locale={locale} messages={messages}>
-          <Providers>{children}</Providers>
+          <AccountStatusProvider>
```

Comment:
I wonder if there restrictions to move `AccountStatusProvider` inside `Providers` ?🤔